integration-test/src/main/java/com/scalar/db/transaction/consensuscommit/TwoPhaseConsensusCommitSpecificIntegrationTestBase.java

```diff
@@ -59,18 +60,43 @@ public abstract class TwoPhaseConsensusCommitSpecificIntegrationTestBase {
   private static final String ANY_ID_1 = "id1";
   private static final String ANY_ID_2 = "id2";
 
-  private TwoPhaseConsensusCommitManager manager;
-  private DistributedStorage storage;
-  private DistributedStorageAdmin admin;
-  private ConsensusCommitAdmin consensusCommitAdmin;
-  private Coordinator coordinator;
-  private String namespace;
+  private TwoPhaseConsensusCommitManager manager1;
+  private TwoPhaseConsensusCommitManager manager2;
```

Comment:
Also separated a `TwoPhaseCommitTransactionManager` instance per table because we assume users execute 2PC transactions with different instances. It's more like an actual situation.